### DISCUSSION TITLE
Get size stats of any app?

### DISCUSSION THREAD

**Discussion by Rudxain** [termux/termux-api#641] [Q&A]
The Android API allows apps with the "App Usage Statistics" permission to request the sizes of code, general data, and cache, of any installed package.

I've noticed that `com.termux` requests that permission, but I haven't found a way to get app size without `adb shell dumpsys diskstats`.

The cmd
```sh
adb shell pm grant com.termux android.permission.DUMP
```
allows execution of `dumpsys` without `shell` privileges, but `diskstats` isn't found as an active service. Perhaps this requires a hack [similar to this](https://github.com/termux/termux-packages/discussions/8292#discussioncomment-5102555)?

If there's no such feature, should I open an issue instead?

**Comment by Cashalot937** [comment]
Give code

**Reply by Rudxain** [reply]
IDK how to do it :(
